### PR TITLE
Quote

### PR DESCRIPTION
**Pull request by UmeshPaliyaal**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import LandingPage from './Pages/LandingPage'
 import NotFound from './NotFound'
+import Quote from './mycomponents/Quote'
```
Comment (yogeshpaliyal) [inline src/App.tsx:6]: Remove this

File: src/components/ui/Quote.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useState , useEffect } from 'react';
+
+function Quote() {
+
```
Comment (yogeshpaliyal) [inline src/components/ui/Quote.tsx:5]: Format this

**Review by yogeshpaliyal — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (UmeshPaliyaal) [reply src/App.tsx]: removed

**Review by yogeshpaliyal — APPROVED** [review]

File: src/components/ui/Quote.tsx (hunk 2 above)
Comment (UmeshPaliyaal) [reply src/components/ui/Quote.tsx]: done

**Pull request merged**